src/visualization/utilities.jl

```diff
@@ -4,6 +4,7 @@
 # See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
 @muladd begin
 
+using LinearAlgebra
```

Comment:
Please put `using` statements at the top of `src/Trixi.jl` and make them explicit, e.g., `using LinearAlgebra: whatever_you_use`. There should already be an entry for `LinearAlgebra`.